transcom/mymove
# B 20165 tio fix page width int

## [B-20165](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A959449)

## Summary

This PR fixes the width of the review weights page for the TIO.
When reviewing weights the TIO shouldn't have to scroll horizontally to see the full page.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as a TIO
2. Click any move in TIO move queue
3. Click "Review Weights"
4. Should see entire page without having to scroll.

### Frontend


## Screenshots
<img width="1247" alt="Screenshot 2024-07-15 at 8 46 15 AM" src="https://github.com/user-attachments/assets/28becfcb-2031-406f-b389-b429fa6dc1e5">
